src/endpoints/disclaimer.js

```diff
@@ -29,10 +30,5 @@ export default new Endpoint({
         order: page.order
       })
     }, null)
-  },
-  mapStateToOptions: (state) => ({
-    language: state.router.params.language,
-    location: state.router.params.location
-  }),
```

Comment:
IMO this should be defined in every endpoint itself, so the endpoint has full control over what's in its options.

Reply:
yeah I will add a withDefaultStateMapper() or similar and remove the defaults